src/dom/DomEvent.DoubleTap.js

```diff
@@ -22,7 +22,7 @@ export function addDoubleTapListener(obj, handler, id) {
 			if ((!Browser.edge) || e.pointerType === 'mouse') { return; }
 			count = _pointersCount;
 		} else {
-			count = e.touches.length;
+			count = e.touches ? e.touches.length : 1;

```

Comment:
This fix is terribly wrong, it could just mask the true root of real issue.

Here we get either `pointer` event, or `touch` event.
And in 'else' path we expect exactly `touch` event, which MUST contain `touches` property.

If there is no such field in your case - it means that something wrong in other parts of code.